packages/components/tests/integration/components/hds/table/index-test.js

```diff
@@ -16,6 +16,11 @@ module('Integration | Component | hds/table/index', function (hooks) {
     assert.dom('[data-test-table]').hasClass('hds-table');
   });
 
+  test('it should render with a CSS class appropriate for the @density value', async function (assert) {
+    await render(hbs`<Hds::Table @density="packed" />`);
+    assert.dom('[data-test-table]').hasClass('hds-table--packed');
```

Comment:
plus, better to use `id` to target the element instead of `data-test`, when `...attributes` is available on the element (also for the other tests in this file).